apps/docs/content/components/Card.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Card
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Card/Card.tsx
+storybook: '/brand/storybook/?path=/story/components-card--playground'
+description: Use the Card component to display information in a compact way and link to other pages.
```

Comment:
Could we make it clear that it links to internal pages and shouldn't be used to link to external ones?

Reply:
```suggestion
description: Use the Card component to display information in a compact way and link to other internal pages.
```